appveyor-linux-binary.yml

```diff
@@ -173,12 +174,20 @@ for:
         - configuration: BuildIntegTesting
 
     test_script:
-      - sh: "pytest -vv tests/integration/buildcmd --ignore=tests/integration/buildcmd/test_build_terraform_applications.py --ignore=tests/integration/buildcmd/test_build_terraform_applications_other_cases.py --json-report --json-report-file=TEST_REPORT-integration-buildcmd.json"
+      - sh: "pytest -vv -n 2 --reruns 3 tests/integration/buildcmd --ignore=tests/integration/buildcmd/test_build_cmd_arm64.py --ignore=tests/integration/buildcmd/test_build_terraform_applications.py --ignore=tests/integration/buildcmd/test_build_terraform_applications_other_cases.py --json-report --json-report-file=TEST_REPORT-integration-buildcmd.json"
```

Comment:
What's the rationale for 2 threads for these tests?

Reply:
These runners have 2 CPUs, I've also tried running with 3 in parallel but they had some flakiness since these tests are usually CPU heavy. 2 in parallel seems like the sweet spot for them